src/test/java/org/dataone/hashstore/ClientTest.java

```diff
@@ -15,12 +10,12 @@
 import java.util.List;
 import java.util.Properties;
 
+import static org.junit.jupiter.api.Assertions.*;
```

Comment:
best practice to call out imports explicitly, rather than use `.*`

Reply:
Noted - thank you!